creme/stats/median.py

```diff
@@ -0,0 +1,107 @@
+import bisect
+import statistics
+import collections
+
+from .. import utils
+
+from . import base
+
+
+__all__ = [
+    'Median',
+    'RollingMedian'
+]
+
+
+class Median(base.Univariate):
```

Comment:
I don't think I like this class... It sure works but it's going to keep growing as time goes on, which is not in the spirit of the library. You should be using `Quantile(0.5)` for the median.

Reply:
@MaxHalford  I did the mentioned changes here #224 . I closed this PR because I was having lot of merge conflicts and rebasing problems. So I closed this PR and opened #224 . We can continue to discuss changes there. Sorry for inconvenience :(